emf/complete/pom.xml

```diff
@@ -0,0 +1,142 @@
+
+        <maven.exec.version>1.6.0</maven.exec.version>
+        <maven.javafx.version>8.8.3</maven.javafx.version>
+        <autoservice.version>1.0-rc2</autoservice.version>
+        <guava.version>20.0</guava.version>
+        <javacsv.version>2.0</javacsv.version>
+
+        <powsybl.core.version>4.2.0</powsybl.core.version>
+        <powsybl.olf.version>0.11.0</powsybl.olf.version>
+        <powsybl.ba.version>1.5.0</powsybl.ba.version>
+        <powsybl.entsoe.version>1.0.0</powsybl.entsoe.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <configuration>
+                    <executable>java</executable>
+                    <arguments>
+                        <argument>-Dpowsybl.config.dirs=${project.build.directory}/classes</argument>
+                        <argument>-classpath</argument>
+                        <argument>powsybl.tutorials.emf.EmfTutorial</argument>
+                    </arguments>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-action-util</artifactId>
+            <version>${powsybl.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-balances-adjustment</artifactId>
+            <version>${powsybl.ba.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-cgmes-conversion</artifactId>
+            <version>${powsybl.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-cgmes-extensions</artifactId>
+            <version>${powsybl.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-commons</artifactId>
+            <version>${powsybl.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-iidm-converter-api</artifactId>
+            <version>${powsybl.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-iidm-mergingview</artifactId>
+            <version>${powsybl.core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-open-loadflow</artifactId>
+            <version>${powsybl.olf.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-config-classic</artifactId>
```

Comment:
This is a bit tricky because we try not to use config-classic on tutorials (in order not to make the user's config interfere). I think you should use config-test and create a config file with `<TO_COMPLETE>` inside (I think the loadflow tutorial looks like it).

Reply:
Just realized there is in fact a config.yml file! I think you should put in there more explicitly that it should be completed (by comments or names such as `to_complete`)